engine-tests/src/tests/eth_connector.rs

```diff
@@ -518,14 +528,45 @@ fn test_ft_transfer_call_without_message() {
     );
     res.assert_success();
```

Comment:
Why do we assert success?

Reply:
This is a quirk of the NEAR system. The transaction succeeds overall even though one promise in it failed.